packages/perseus-core/src/widgets/plotter/index.ts

```diff
@@ -0,0 +1,45 @@
+import type {PerseusPlotterWidgetOptions} from "../../data-schema";
+import type {WidgetLogic} from "../logic-export.types";
+
+export type PlotterDefaultWidgetOptions = Pick<
+    PerseusPlotterWidgetOptions,
+    | "scaleY"
+    | "maxY"
+    | "snapsPerLine"
+    | "correct"
+    | "starting"
+    | "type"
+    | "labels"
+    | "categories"
+    | "picSize"
+    | "picBoxHeight"
+    | "plotDimensions"
+    | "labelInterval"
+    | "picUrl"
+>;
+
+const defaultWidgetOptions: PlotterDefaultWidgetOptions = {
+    scaleY: 1,
+    maxY: 10,
+    snapsPerLine: 2,
+    correct: [1],
+    starting: [1],
+
+    type: "bar",
+    labels: ["", ""],
+    categories: [""],
+
+    picSize: 30,
+    picBoxHeight: 36,
+    plotDimensions: [275, 200],
```

Comment:
Woah, that's some random size. 😂  Our graphs are usually 400x400. 

No action needed.